src/main/java/io/github/darkkronicle/kronhud/gui/hud/ActionBarHud.java

```diff
@@ -56,7 +56,7 @@ public void renderPlaceholder(MatrixStack matrices) {
         matrices.push();
         renderPlaceholderBackground(matrices);
         scale(matrices);
-        client.textRenderer.draw(matrices, placeholder,  (float)getPos().x() + Math.round((float) width /2) - (float) client.textRenderer.getWidth(placeholder) /2, (float)getPos().y() + 3, textColor.getColor().color());
```

Comment:
Why not have add the textColor option?

Reply:
I removed it again since it overrode the vanilla coloring (which also includes the fade-out animation). I think a good solution would be to add another option whether the color value should be overridden or not.

Reply:
You could just use the vanilla opacity if the colour is set to full alpha.

Reply:
btw.. You seem to have a similar Project as I do..